src/components/ComposedModal/ComposedModal.jsx

```diff
@@ -131,8 +133,6 @@ export const ComposedModalPropTypes = {
   invalid: PropTypes.bool, // eslint-disable-line react/boolean-prop-naming
   /** Callback to submit the dialog/form */
   onSubmit: PropTypes.func,
-  /** switchc to render footer buttons or not (carbon consistency) */
-  passiveModal: PropTypes.bool,
```

Comment:
This will cause a breaking change. Is there any way we can deprecate this first, then remove in next major release?

Reply:
Otherwise this looks good to go.

Reply:
ok repushed with false rather than removing